frontend/javascripts/oxalis/view/right-border-tabs/skeleton_tab_view.js

```diff
@@ -649,7 +656,7 @@ class SkeletonTabView extends React.PureComponent<Props, State> {
   }
 
   getSelectedTreesAlert = () =>
-    this.state.selectedTrees.length > 0 ? (
+    this.state.selectedTrees.length > 0 && this.props.allowUpdate ? (
```

Comment:
Why should this alert be disabled, it only helps to deselect the selection, doesn't it? If anything, shouldn't the tree selection itself be disabled, instead? However, I don't think that's important since it cannot be used to modify anything as far as I can see.